src/ert/analysis/_es_update.py

```diff
@@ -238,8 +238,9 @@ def _save_temp_storage_to_disk(
                 assert isinstance(_matrix, xr.DataArray)
                 target_fs.save_parameters(key, realization, _matrix.to_dataset())
             elif isinstance(config_node, Field):
```

Comment:
maybe combine Field and Surface handling as the code is identical?
elif isinstance(config_node, (Field, SurfaceConfig)):

Reply:
Good catch, thanks 👍